samples/nrf9160/gps/src/main.c

```diff
@@ -31,16 +31,27 @@
 #ifdef CONFIG_BOARD_NRF9160DK_NRF9160NS
 #define AT_MAGPIO      "AT\%XMAGPIO=1,0,0,1,1,1574,1577"
 #ifdef CONFIG_GPS_SAMPLE_ANTENNA_ONBOARD
-#define AT_COEX0       "AT\%XCOEX0=1,1,1570,1580"
+#define AT_COEX0       "AT\%XCOEX0=1,1,1565,1586"
 #elif CONFIG_GPS_SAMPLE_ANTENNA_EXTERNAL
 #define AT_COEX0       "AT\%XCOEX0"
 #endif
+#endif /* CONFIG_BOARD_NRF9160DK_NRF9160NS */
+
+#ifdef CONFIG_BOARD_THINGY91_NRF9160NS
+#define AT_MAGPIO      "AT\%XMAGPIO=1,1,1,7,1,746,803,2,698,748,2,1710,2200," \
+			"3,824,894,4,880,960,5,791,849,7,1565,1586"
+#ifdef CONFIG_GPS_SAMPLE_ANTENNA_ONBOARD
+#define AT_COEX0       "AT\%XCOEX0=1,1,1565,1586"
+#elif CONFIG_GPS_SAMPLE_ANTENNA_EXTERNAL
+#define AT_COEX0       "AT\%XCOEX0"
 #endif
+#endif /* CONFIG_BOARD_THINGY91_NRF9160NS */
 
-static const char            update_indicator[] = {'\\', '|', '/', '-'};
-static const char            at_commands[][31]  = {
+static const char		update_indicator[] = {'\\', '|', '/', '-'};
+static const char		at_commands[][sizeof(AT_MAGPIO)]  = {
```

Comment:
Why isn't it actually array of pointers? A few bytes in flash could be saved then.

Reply:
Good point, the array size blew up quite a bit on Thingy:91 in particular. I'll do that change along with others later in the review, if there are any.